vespa-http-client/src/main/java/com/yahoo/vespa/http/client/core/api/FeedClientImpl.java

```diff
@@ -60,14 +61,13 @@ public void stream(String documentId, CharSequence documentData, Object context)
     @Override
     public void close() {
         Instant lastResultReceived = Instant.now();
```

Comment:
Strictly this should be called something else now.

Reply:
Yes :(